README.md

```diff
@@ -907,6 +907,7 @@ See [go-hardware](https://github.com/rakyll/go-hardware) for a comprehensive lis
 * [medeina](https://github.com/imdario/medeina) - Medeina is a HTTP routing tree based on HttpRouter, inspired by Roda and Cuba.
 * [mux](https://github.com/gorilla/mux) - A powerful URL router and dispatcher for golang.
 * [neo](https://github.com/ivpusic/neo) - Neo is minimal and fast Go Web Framework with extremely simple API.
+* [Ozzo](https://github.com/go-ozzo) - A simple Go web toolkit written by author of legenary yii PHP framwork.
```

Comment:
The word "legenary" is misspelled.

It would be better to use more neutral, objective language, like "written by author of yii PHP framwork."